.circleci/config.yml

```diff
@@ -229,6 +229,17 @@ jobs:
       - store_artifacts:
           path: /home/circleci/project/target/inspection-results
 
+  no-merge-commits:
+    docker:
+      - image: cimg/base:2022.10
+    steps:
+      - checkout
+      - run:
+          name: Check for merge commits
+          command: |
+            export PR_HEAD_SHA=$CIRCLE_SHA1
```

Comment:
renamed this variable to be CI agnostic, and we cannot do this in `environment` block since it is built-in variable